simphony/io/h5_particles.py

```diff
@@ -133,11 +138,16 @@ def name(self, value):
 
     @property
     def data(self):
-        raise NotImplementedError()
+        if len(self._data) == 1:
+            return self._data[0]
+        return DataContainer()
```

Comment:
*suggestion*: For consistency I would use the if...else block similar to the ``setter`` method